ui/src/pages/frontsite/components/Document/DocumentInfo.js

```diff
@@ -23,6 +24,49 @@ export class DocumentInfo extends Component {
     return null;
   }
 
+  renderISBN() {
+    const identifiersISBN = this.metadata.alternative_identifiers
+      ? this.metadata.alternative_identifiers.filter(
+          identifier => identifier.scheme === 'ISBN'
```

Comment:
Could you move the ISBN to be in the identifiers? I know it is now in the  select under `alternative_identifiers` but it was not intended to be, as it is specific to the domain, therefore not something additional but rather crucial.
In order to do that you will have to change one of the vocabularies JSON files to move it to `identifiers`, similarly for series.
As a results this field will be taken here from the `identifiers` in the end, like the DOI below